types/node/index.d.ts

```diff
@@ -1188,7 +1182,7 @@ declare module "http" {
     }
 
     // https://github.com/nodejs/node/blob/master/lib/_http_server.js#L108-L256
-    export class ServerResponse extends OutgoingMessage {
```

Comment:
Why remove `export` here? third party library authors maybe need those.

Reply:
Disregard this one, see my other comment, this work perfectly fine, as mentioned, `export` was redundant here.